requirements.txt

```diff
@@ -2,3 +2,4 @@ pytest
 pytest-html
 pytest-cov
 flake8
+pyopenssl
```

Comment:
does pyopenssl really need to be here?  seems like the extras_require list is the more appropriate way

Reply:
It's in the extras_require, but extras_require is only used when installing modules from other modules (apparently). This is here so that the CI unit testing can reference these modules and get the code coverage up.